src/NuGet.Core/NuGet.Protocol/EnhancedHttpRetryHelpers.cs

```diff
@@ -2,103 +2,110 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using NuGet.Common;
 
 namespace NuGet.Protocol
 {
     internal class EnhancedHttpRetryHelper
     {
-        private readonly IEnvironmentVariableReader _environmentVariableReader;
-        internal const string ExperimentalRetryEnabledEnvVarName = "NUGET_ENABLE_EXPERIMENTAL_HTTP_RETRY";
-        internal const string ExperimentalRetryTryCountEnvVarName = "NUGET_EXPERIMENTAL_MAX_NETWORK_TRY_COUNT";
-        internal const string ExperimentalRetryDelayMsEnvVarName = "NUGET_EXPERIMENTAL_NETWORK_RETRY_DELAY_MILLISECONDS";
+        /// <summary>
+        /// The default delay in milliseconds between retries.
+        /// </summary>
+        public const int DefaultDelayMilliseconds = 1000;
 
-        public EnhancedHttpRetryHelper(IEnvironmentVariableReader environmentVariableReader)
-        {
-            _environmentVariableReader = environmentVariableReader ?? throw new ArgumentNullException(nameof(environmentVariableReader));
-        }
+        /// <summary>
+        /// The default value indicating whether or not enhanced HTTP retry is enabled.
+        /// </summary>
+        public const bool DefaultEnabled = true;
```

Comment:
What's the benefit of a public constant for whether something is enabled or what its default values are?  Could callers not just Get() the existing stuff?

Reply:
I only added this to update unit tests to expect the current default.  If we ever decided to turn this off by default, we could update this one constant and not have to update the unit tests' expectations (unlikely in this case but its just a constant to not a big deal).